src/core/Akka.NodeTestRunner/Program.cs

```diff
@@ -66,15 +91,80 @@ static void Main(string[] args)
                             specFail.FailureExceptionTypes.Add(ex.GetType().ToString());
                             specFail.FailureMessages.Add(ex.Message);
                             specFail.FailureStackTraces.Add(ex.StackTrace);
+                            _logger.Tell(specFail.ToString());
                             Console.WriteLine(specFail);
+
+                            //make sure message is send over the wire
+                            FlushLogMessages();
                             Environment.Exit(1); //signal failure
+                            return 1;
```

Comment:
Added this for Mono / Unix compatibility in the future. We got burned by this issue on NBench so I thought it best to preemptively add it here. Effectively a no-op on Windows.